src/components/menus/TertiaryNav/TertiaryNav.scss

```diff
@@ -0,0 +1,83 @@
+@import '../../../styles/_partials/index';
+
+.TertiaryNavContainer {
+	display: flex;
+	flex-direction: row;
+	flex: 1; // this assumes its parent is a flex element
+	overflow-y: auto;
+	@include theme-background-white-else-graydark;
+
+	// needs to be global so any element, outside this library, can use it
+	:global(.TertiaryNavChild__Error) {
+		color: $red;
+	}
+
+	// reset unordered lists
+	ul,
+	ul li,
+	ul ul li {
+		margin: 0;
+		padding: 0;
+		text-indent: 0;
+		list-style-type: none;
+	}
+
+	.TertiaryNav {
+		width: 200px;
+		padding: 16px 12px;
+		overflow-y: auto;
+		@include theme-border-right;
+	}
+
+	.TertiaryNavItem {
+		a {
+			position: relative;
+			display: inline-block;
+			width: 100%;
+			height: 36px;
+			line-height: 36px;
+			padding: 1px 20px 0;
+			border-radius: 4px;
+			color: $gray75;
+			font-weight: 500;
+			font-size: 14px;
+			@include tracking(40);
```

Comment:
Lost in moving this from .sass to .scss is the removal of this line which can more clearly be seen in [this commit](https://github.com/getflywheel/local-components/pull/149/commits/02fa75143861ee2269d91d92e138b12de450ba01):
```text-transform: uppercase;```

Reply:
This is the only change in this file btw.